packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -232,18 +239,22 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
         overflows[overflows.length - 1] = 'and ${overflows[overflows.length - 1]}';
         overflowText = overflows.join(', ');
     }
+    // TODO(jacobr): add the overflows in pixels as structured data so they can
+    // be visualized in debugging tools.
     FlutterError.reportError(
       FlutterErrorDetailsForRendering(
         exception: 'A $runtimeType overflowed by $overflowText.',
```

Comment:
Should this line be the error's summary?

Reply:
Wow. We shouldn't have exceptions that are strings. @pq we should probably add a lint to catch this case although it is low priority because the good news is this is the only place in the framework where that mistake was made.
<img width="1190" alt="Screen Shot 2019-04-18 at 4 17 38 PM" src="https://user-images.githubusercontent.com/1226812/56396829-a420b700-61f5-11e9-883b-1e4fe3b37cf5.png">
Fixed this to construct a proper FlutterError object with a summary. The reason for this bug is the Dart type system lacks a single base class for all exceptions.